content/en/docs/ops/best-practices/observability/index.md

```diff
@@ -146,6 +146,10 @@ Add the following job to your configuration:
     'match[]':
     - '{__name__=~"workload:(.*)"}'
     - '{__name__=~"pilot(.*)"}'
+    - '{__name__=~"envoy(.*)"}'
+    - '{__name__=~"go(.*)"}'
+    - '{__name__=~"istio(.*)"}'
```

Comment:
We don't want istio metrics directly. That is the point of the the `workload` prefix in this section of the document.

This is the same thing below.

Reply:
Alright, removing istio below makes perfect sense.
I'll update the PR with the removal and mention these between {{< tip >}}  {{< /tip >}} tags. Thank you for picking this up!